.github/workflows/pr-test.yml

```diff
@@ -5,7 +5,6 @@ on:
       - main
   pull_request:
 
-
```

Comment:
Looks like no change.

Reply:
ahh. I was planning to add branch restriction here as well. maybe that line got removed. no worries

Reply:
hmm - it might be good to have symmetry with the `pull_request_target` workflow, but on the other hand, raising a PR against a non-main branch is a fine practice, so no harm done by leaving it unfiltered.